pubsub/azurepubsub/azurepubsub.go

```diff
@@ -435,6 +450,11 @@ func (s *subscription) SendAcks(ctx context.Context, ids []driver.AckID) error {
 		return nil
 	}
 
+	// Invoke AckFunc is provided (at-most-once mode).
+	if s.opts != nil && s.opts.AckFuncForReceiveAndDelete != nil {
+		s.opts.AckFuncForReceiveAndDelete()
+	}
+
```

Comment:
On line 520, you should return the ackFunc.

Reply:
done